_includes/card-purchase.md

```diff
@@ -121,17 +121,17 @@ Content-Type: application/json
 | {% icon check %} | └➔&nbsp;`operation`               | `string`      | Determines the initial operation, that defines the type card payment created.<br> <br> `Purchase`. Used to charge a card. It is followed up by a capture or cancel operation.<br> <br> `Recur`.Used to charge a card on a recurring basis. Is followed up by a capture or cancel operation (if not Autocapture is used, that is).<br> <br>`Payout`. Used to deposit funds directly to credit card. No more requests are necessary from the merchant side.<br> <br>`Verify`. Used when authorizing a card withouth reserveing any funds.  It is followed up by a verification transaction. |
 | {% icon check %} | └➔&nbsp;`intent`                  | `string`      | `Authorization`. Reserves the amount, and is followed by a [cancellation][cancel] or [capture][capture] of funds.<br> <br> `AutoCapture`. A one phase option that enable capture of funds automatically after authorization.                                                                                                                                                                                                                                                                                                                                                              |
-|                  | └➔&nbsp;`paymentToken`            | `string`      | If you put in a paymentToken here, the payment page will preload the stored payment data related to the `paymentToken` and let the consumer make a purchase without having to enter all card data. This is called a "One Click" purchase.                                                                                                                                                                                                                                                                                                                                                 |
+|                  | └➔&nbsp;`paymentToken`            | `string`      | If you enter a paymentToken here, the payment page will preload the stored payment data related to the `paymentToken`, and the payer can complete the purchase without having to enter all card data. This is called a "One Click" purchase.                                                                                                                                                                                                                                                                                                                                                 |
```

Comment:
This sentence reads very strange to me and "the payment page will reload" is blatantly false, isn't it?

Reply:
It says preload, not reload :) If you enter a paymentToken which has been "activated" by a previous payment, the card will appear masked as a stored card, with the CVC as the only field that needs input.

But I agree that the wording is a bit awkward. I can re-phrase it for sure.

Reply:
If I understood your question correctly?

Reply:
I see. "Prefilled" is a term used elsewhere, so perhaps that's better?

Reply:
Will you rephrase it in this PR or in another?

Reply:
I'll fix it here 👍

Reply:
Better? :)

Reply:
Much! 👏